extensions/neo4j/runtime/src/main/java/io/quarkus/neo4j/BossGroup.java

```diff
@@ -0,0 +1,14 @@
+package io.quarkus.neo4j;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+
+import javax.inject.Qualifier;
+
+/**
+ * @author Michael J. Simons michael.simons@neo4j.com
+ */
+@Qualifier
+@Retention(RetentionPolicy.RUNTIME)
+public @interface BossGroup {
```

Comment:
I'm not sure we need this?

Reply:
See above, as our driver shades Netty, all it's extensions and mods are applicable as well.

Reply:
I do have a dependency, see

`io.quarkus.neo4j.deployment.NettyProcessor`

It's an adapted version of the `NettyProcessor`, that has been adapted to use the shaded packages.

Reply:
Sure. But do you use the beans generated by `NettyProcessor` somewhere? I don't see them consumed in your work but, as I said, I don't know Netty well so I might miss how these things are wired.

Reply:
I just removed it and it still works… For whatever reason, I ended up in a situation before where things didn't work in the end. Must have been confused by too many poms and files open exception. They will be gone in a sec. Thanks for not letting it go :)